node-fs-writefile/cp.js

```diff
@@ -0,0 +1,27 @@
+import * as fs from 'node:fs/promises';
+
+const [, , original, copy] = process.argv;
+
+async function readOriginal(originalPath) {
+  try {
+    const data = await fs.readFile(originalPath, 'utf-8');
```

Comment:
You could combine these two lines if you wanted:
`return await fs.readFile(...);`